src/fluent-bit.c

```diff
@@ -71,6 +71,7 @@ extern void win32_started(void);
 
 flb_ctx_t *ctx;
 struct flb_config *config;
+int ch_context_signal;
```

Comment:
Just for the sake of correctness change this to be a `flb_pipefd_t` which will transparently be aliased to `int` in the relevant systems.